test/5.public.HtmlChecker.js

```diff
@@ -302,7 +302,44 @@ describe("PUBLIC -- HtmlChecker", function()
 				}
 			}).scan(htmlString, baseUrl);
 		});
+
+		it("excludeExternalLinks = false", function(done)
+		{
+			var results = [];
+			
+			var htmlString = '<a href="http://example.com">link1</a>';
```

Comment:
I'm getting really picky here, but to be consistent with the other tests containing only one link, please change the text to "link" instead of "link1"